src/app/game.ts

```diff
@@ -0,0 +1,373 @@
+import {
+  Building,
+  Unit,
+  House,
+  Wall,
+  Warehouse,
+  Goblin,
+  Knight,
+  Goldmine,
+} from './components'
```

Comment:
Hmm... maybe we could remove building and units from components folder. Remove components folder. Add pop up to utils. If you have different opinion we can leave it

Reply:
Why to utils? These are not utils classes. If we don't want to keep every files for one feature, e.g. under goldmine folder all ts, css etc. files then we can move all buildings and units from components folder and put them in models folder

Reply:
Ok